### DISCUSSION TITLE
Local outline?

### DISCUSSION THREAD

**Discussion by Bence98007** [outline/outline#3134] [Self Hosting]
Hi there!

Just wanted to ask if its possible to run outline in docker on a local network, without public access? I saw that the authentication needs to be something third-party (slack, google) but i dont want to publish my documentation just because to enable login via slack/google.
I saw the feature request about allowing email/pass auth, but there is no exact time or respone from the devs about it.
I know I know, just buy the enterprise verison to get it cloud hosted and I shouldnt worry anymore, but I dont want that.

**Comment by tommoor** [comment]
It is possible to use any OIDC compatible provider for authentication, there are several including Keybase that you can host on an internal network.

**Reply by Bence98007** [reply]
Sounds good, I missed that from the documentation. Thanks for the info!
I have one more question, idk if I should open a new discussion again, but:
The SMTP throws back SSL error. I want to use Office365 to send out emails, is it possible to do so?
What should I put in the SMTP_TLS_CIPHERS? This is my config for the SMTP settings:
# Emails configuration (optional)
SMTP_HOST=smtp.office365.com
SMTP_PORT=587
SMTP_USERNAME=user@domain.com
SMTP_PASSWORD=12345
SMTP_FROM_EMAIL=user@domain.com
SMTP_REPLY_EMAIL=user@domain.com
SMTP_TLS_CIPHERS=
SMTP_SECURE=true

Thanks for helping!
